src/plugins/ui/sheetview.ts

```diff
@@ -209,7 +209,7 @@ export class SheetViewPlugin extends UIPlugin {
         break;
       case "UPDATE_CELL":
         // update cell content or format can change hidden rows because of data filters
-        if ("content" in cmd || "format" in cmd) {
+        if ("content" in cmd || "format" in cmd || cmd.style?.fontSize !== undefined) {
```

Comment:
It's unclear to me whose responsibility it is to reset the viewports. This plugin or data filters/auto-resize